src/main/java/vg/civcraft/mc/civmodcore/CivStrings.java

```diff
@@ -0,0 +1,6 @@
+package vg.civcraft.mc.civmodcore;
+
+public final class CivStrings {
```

Comment:
This class seems pretty pointless. Why not just put that string in the class it's used?